src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  let activeContent = '';
+  const isIdValid = tabs.some(item => item.id === activeTabId);
+
+  const preparedTabs = tabs.map((tab, index) => {
+    const { id, content, title } = tab;
+    const isTabActive = id === activeTabId;
+
+    if (isTabActive) {
+      activeContent = content;
+    }
```

Comment:
you can just try to find(using the find method) your tab by activeTabId, instead of the some and map methods